extension/intents/clipboard/contentScript.js

```diff
@@ -108,14 +108,17 @@ this.contentScript = (function() {
 
   types.copyImage = function() {
     const img = document.querySelectorAll("img");
-    const imgHeight = [];
-    img.forEach(function(element) {
-      if (isInViewport(element)) {
-        imgHeight.push(element.clientHeight);
+    let maxHeightIndex = 0;
+    const isValidSrc = src =>
+      src.startsWith("data:") || /png|jpe?g/.test(src.split(".").pop());
```

Comment:
URLs can be anything, it's best not to check the text of the URL to try to determine the type of the file. Anything in an `<img>` should be an image of some sort.